src/sbuf/SBuf.cc

```diff
@@ -262,7 +262,10 @@ SBuf::vappendf(const char *fmt, va_list vargs)
         // not enough space on the first go, we now know how much we need
         requiredSpaceEstimate = sz*2; // TODO: tune heuristics
         space = rawSpace(requiredSpaceEstimate);
-        sz = vsnprintf(space, spaceSize(), fmt, vargs);
+        va_list ap2;
+        va_copy(ap2, vargs);
```

Comment:
> When resizing after the first vsnprintf, we were reusing the caller's
va_list,

I do not see **re**use. I see first-time use. Before resizing, we were using a _copy_ of caller's va_list (stored in `ap`). Am I missing something?

Reply:
You are not missing something; I got the variables mixed up of the first `va_copy` call.